bundles/plugins/org.bonitasoft.studio.refactoring/src/org/bonitasoft/studio/refactoring/ui/CompareScript.java

```diff
@@ -75,56 +63,31 @@ public String getType() {
     }
 
     @Override
-    public InputStream getContents() throws CoreException {
-        if (content instanceof Expression) {
-            return new ByteArrayInputStream(((Expression) content).getContent().getBytes());
-        }
-        return null;
-    }
-
-    public Expression getContentExpression() {
-        if (content instanceof Expression) {
-            return (Expression) content;
-        }
-        return null;
-    }
-
-    @Override
     public Object[] getChildren() {
         return children.toArray();
     }
 
-    public void addChild(Object child) {
+    public void addChild(final Object child) {
         children.add(child);
     }
 
     @Override
     public boolean isEditable() {
-        return true;
+        return false;
```

Comment:
Why the original script was editable in the compare view ? It seems not a valid use as we do not update the model with "left changes"